### PR TITLE
Raise error in case of booting from volume and from image

### PR DESCRIPTION
**Pull request by shiramax**
Fixes https://projects.theforeman.org/issues/26658

### CODE REVIEW

File: app/models/foreman_kubevirt/kubevirt.rb [hunk 1]
```diff
@@ -150,6 +150,13 @@ def create_vm(args = {})
         volume.type = 'containerDisk'
         volumes << volume
       end
+      is_bootable = false
+      volumes_attributes.each do |_, v|
```
Comment (masayag) [inline app/models/foreman_kubevirt/kubevirt.rb]: you can simplified that to:
`raise ::Foreman::Exception.new(N_('It is not possible to set a bootable volume and image based provisioning.')) if args.key?("image_id") && volumes_attributes.any? { |_,v| v["bootable"] == "true" }`

Comment (masayag) [reply app/models/foreman_kubevirt/kubevirt.rb]: also, you can pull this out to a method and add a test to cover it.

**Comment by shiramax** [comment]
@masayag done, please review again

File: app/models/foreman_kubevirt/kubevirt.rb [hunk 2]
```diff
@@ -150,6 +156,8 @@ def create_vm(args = {})
         volume.type = 'containerDisk'
         volumes << volume
       end
+      verify_booting_from_image_is_possible(args)
```
Comment (masayag) [inline app/models/foreman_kubevirt/kubevirt.rb]: If we'll move the validation to the image_provision block (line 153) , we can remove the check for the existence image_id key.

what do you think ?

Comment (shiramax) [reply app/models/foreman_kubevirt/kubevirt.rb]: @masayag , done :)

File: app/models/foreman_kubevirt/kubevirt.rb [hunk 3]
```diff
@@ -142,6 +148,7 @@ def create_vm(args = {})
       # Add image as volume to the virtual machine
       image_provision = args["provision_method"] == "image"
       if image_provision
+        verify_booting_from_image_is_possible(args)
```
Comment (masayag) [inline app/models/foreman_kubevirt/kubevirt.rb]: so if we don't anything else from args, except of volumes_attributes, we can pass to this method `volumes_attributes` that was set to a variable on line 145.

**Comment by shiramax** [comment]
@masayag done.

**Pull request merged**